### ISSUE TITLE
One dimensional systems

### ISSUE THREAD

**Issue by bfocassio** [Z2PackDev/Z2Pack#90]
How can one use z2pack for topological classification of 1D systems (e.g. SSH model) ?

**Comment by greschd** [comment]
In some cases, the polarization / sum of Wannier charge centers is quantized to {0, 0.5} - that gives a topological classification. A single line can be calculated with `z2pack.line.run`.

**Comment by bfocassio** [comment]
Then I would simply do:

```
system = z2pack.tb.System( ... )
result = z2pack.line.run(system=system,line=lambda t: [t, 0, 0])
pol = result.pol
```
For the SSH model:
for `t=1` and `dt=0.2` : `pol=0.0`
for `t=1` and `dt=-0.2`: `pol=0.5`

**Comment by greschd** [comment]
Yes, exactly. You can also define the model directly as one-dimensional if that's more convenient.

For a full (but somewhat raw) example, see the attached notebook.

[ssh_model.zip](https://github.com/Z2PackDev/Z2Pack/files/4981382/ssh_model.zip)
